mirage/views/resource.ts

```diff
@@ -1,10 +1,22 @@
 import { HandlerContext, Request, Response, Schema } from 'ember-cli-mirage';
 
+export function createResource(this: HandlerContext, schema: Schema) {
+    const attrs = {
+        ...this.normalizedRequestAttrs('resource'),
+        finalized: false,
+    };
```

Comment:
Could you have this set `resourceType` to `undefined` to match BE functionality?